Framework/DataObjects/test/WorkspaceCreationTest.h

```diff
@@ -172,6 +172,15 @@ class WorkspaceCreationTest : public CxxTest::TestSuite {
     check_instrument(*ws);
   }
 
+  void test_create_parent_without_logs() {
+    const auto parent = create<Workspace2D>(m_instrument, make_indices(),
+                                            Histogram(BinEdges{1, 2, 4}));
+    const auto ws = createWithoutLogs<Workspace2D>(*parent);
+    check_indices(*ws);
+    check_zeroed_data(*ws);
+    check_instrument(*ws);
```

Comment:
You are not testing anything here? I could replace line 178 by `create` and the test would still pass, I think?